--- origsrc/libchewing-0.3.3/Makefile.am	2011-02-27 06:49:22.000000000 -0600
+++ src/libchewing-0.3.3/Makefile.am	2011-10-03 01:15:25.597320600 -0500
@@ -34,15 +34,13 @@ pkgconfigdir = $(libdir)/pkgconfig
 
 pkgconfig_DATA = chewing.pc
 
-chewing_headers = \
+chewing_includedir = $(includedir)/chewing
+chewing_include_HEADERS = \
 	include/chewingio.h \
 	include/mod_aux.h \
 	include/chewing.h \
 	include/global.h
 
-install-data-local:
-	$(mkinstalldirs) $(DESTDIR)/$(prefix)/include/chewing
-	$(INSTALL_DATA) $(chewing_headers) $(DESTDIR)/$(prefix)/include/chewing
 install-manpages:
 	$(mkinstalldirs) $(DESTDIR)/$(prefix)/share/man/man3
 	$(INSTALL_DATA) man/man3/*.3 $(DESTDIR)/$(prefix)/share/man/man3
